client/plugins/os_updates.py

```diff
@@ -179,9 +181,21 @@ def status(current, updates):
             return status(True, False)
         log.error('Call to pacman -Qu failed. Output: {}',
                   e.output.decode('utf8'))
-        return status(True, 'unknown')
+        return status(True, True, 'unknown')
```

Comment:
this isn't new to this PR, but can we pass these by keyword, I have to keep checking what arg is what.